Assets/Plugins/FMOD/Resources/FMODStudioSettings.asset

```diff
@@ -462,8 +462,8 @@ MonoBehaviour:
   switchSettingsMigration: 1
   HasSourceProject: 1
   HasPlatforms: 1
-  sourceProjectPath: 
-  sourceBankPath: 
+  sourceProjectPath: MFPR-FMOD_Repo/MFPR.fspro
```

Comment:
This is optional. Unless people using this submodule want to test and update the audio live, we only need to include the banks for implementation